libmscore/album.cpp

```diff
@@ -179,7 +257,7 @@ bool Album::scoreInActiveAlbum(MasterScore *score)
     if (!activeAlbum)
```

Comment:
Hmm, this method should probably be a `const` method, since it should never change anything about the `Album`.